docs/Getting Started.md

```diff
@@ -0,0 +1,59 @@
+# Getting Started
+
+## Clone Serai
+```
+git clone https://github.com/serai-dex/serai.git
+```
+
+## Build and Run Serai
+
+### Install Rust
+
+#### Linux
+```
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+#### Windows
+
+Use WSL2 and Linux instructions.

```

Comment:
Do we care to even mention Windows at this point? We can just officially support Linux and if someone says "But I run Windows" say "WSL2 at your own risk". I know, rolling stone from my previous comments. I just truly don't care about Windows development for server-software.